tc/sgx/trusted_worker_manager/enclave/work_order_preprocessed_keys.cpp

```diff
@@ -0,0 +1,185 @@
+/* Copyright 2020 Intel Corporation
```

Comment:
Can we rename this file to work_order_preprocessed_keys_wpe.cpp or work_order_keys_wpe.cpp

Reply:
Done